src/python/WMCore/WMSpec/WMTask.py

```diff
@@ -1327,6 +1327,55 @@ def getConfigCacheIDs(self):
                 IDs.append(ID)
         return IDs
 
+    def physicsTaskType(self):
```

Comment:
I would suggest renaming it to `getPhysicsTaskType`

Reply:
This was my initial thought, but I called it `physicsTaskType`  just for consistency with the general task type, which is called `taskType` and not `getTaskType`. Should we rename both to add the `get`?

Reply:
I would update only this one. The `taskType` is likely used in gazillions of places and we could easily miss a place or two and inject a bug.